modules/ROOT/pages/dw-core-functions-match.adoc

```diff
@@ -6,14 +6,14 @@ endif::[]
 
 
 [[match1]]
-== match&#40;String, Regex&#41;: Array<String&#62;
+== match&#40;text: String, matcher: Regex&#41;: Array<String&#62;
 
 Uses a Java regular expression (regex) to match a string and then separates it into
 capture groups. Returns the results in an array.
 
 
 Note that you can use `match` for pattern matching expressions that include
-https://docs.mulesoft.com/mule-runtime/4.3/dataweave-pattern-matching[case
+https://docs.mulesoft.com/mule-runtime/latest/dataweave-pattern-matching[case
```

Comment:
TODO: latest prob OK but if we ever remove a file, the link won't work. We have to use HTTPS so that these links can be single-sourced to other places.